### PR TITLE
minimize window if already focused

### LINKED ISSUE #27: Pls ass option to minimize if application is focused already on run
Pls add option to minimize if application is focused already on run 
something like 
--minimize_if_focused

For example to run calculator
i click hotkey, calculator started, when i finish calculations i want to minimize it with same hotkey

### PR DESCRIPTION
**Pull request by matklad**
~~closes #12~~

tested on plasma, seems to work

EDIT: 

closes #27 
based on #12

### CODE REVIEW

**Comment by matklad** [comment]
FYI, I've packaged jumpapp for NixOs: https://github.com/NixOS/nixpkgs/pull/57893

File: jumpapp [hunk 1]
```diff
@@ -10,31 +10,33 @@ Otherwise, launch COMMAND (with opitonal ARGs) to start the application.
...
+  -m -- if a single window is already open and in focus - minimize it
   -n -- do not fork into background when launching COMMAND
   -p -- always launch COMMAND when ARGs passed
         (see Argument Passthrough in man page)
   -L -- list matching windows for COMMAND and quit
   -t NAME -- process window has to have NAME as the window title
   -c NAME -- find window using NAME as WM_CLASS (instead of COMMAND)
   -i NAME -- find process using NAME as the command name (instead of COMMAND)
-  -w -- only find the applications in the current workspace
+k  -w -- only find the applications in the current workspace
```
Comment (Rublis) [inline jumpapp]: some weird "k" slipped in! :)

File: jumpapp [hunk 2]
```diff
@@ -10,6 +10,7 @@ Otherwise, launch COMMAND (with opitonal ARGs) to start the application.
 Options:
   -r -- cycle through windows in reverse order
   -f -- force COMMAND to launch if process found but no windows found
+  -m -- if a single window is already open and in focus - minimize it
```
Comment (mkropat) [inline jumpapp]: Makes sense 👍

**Comment by matklad** [comment]
Added a test!

The testing setup is awesome: it was very easy to figure out how to add tests, despite the fact that I've never tested bash scripts before :)

**Comment by matklad** [comment]
btw, https://github.com/mkropat/jumpapp/issues/27#issuecomment-473054964 suggest making `-m` the default behavior. This makes sense, because currently, if there's only a single window and it is active, jumpapp does nothing. But, because jumpapp is usually called from xbindkeysrc, I think requiring a flag is not a big deal here, and it helps to preserve existing behavior.

**Comment by mkropat** [comment]
Tests look good. Change looks good 👍 

I thought I'd find the time to test the change but I never seem to find the free time.

I've merged it without testing it myself, but I figure on the low chance there's some issue, someone can report it.

Thanks again for the PR.

**Pull request merged**